### PR TITLE
Fix order show page

### PR DESCRIPTION
**Pull request by ButeraMV**
LIST OF CHANGES:
- Added more info to order page
- Capitalized the statuses
-
-

THINGS I'M UNSURE ON:
-
-
-

RSPEC GREEN?
YES

*MENTION GROUP MEMBER TO REVIEW PR*

### CODE REVIEW

File: app/models/order.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Order < ApplicationRecord
   has_many :order_items
   has_many :items, through: :order_items
 
-  enum status: ["ordered", "paid", "cancelled", "completed"]
+  enum status: ["Ordered", "Paid", "Cancelled", "Completed"]
 
   def add(item_hash)
     item_hash.each do |item, item_quantity|
```
Comment (blsrofe) [inline app/models/order.rb:10]: This looks a lot better!

**Review by blsrofe — APPROVED** [review]

File: app/views/orders/show.html.erb [hunk 2]
```diff
@@ -2,21 +2,24 @@
   <br>
   <h1><strong><%= "Order #{@order.id}" %></strong></h1>
   <br>
+      <p>Placed on: <%= @order.created_at %></p>
+      <p>Order Status: <%= @order.status %></p>
```
Comment (snayrouz) [inline app/views/orders/show.html.erb:6]: thanks for adding these

File: app/views/orders/show.html.erb [hunk 3]
```diff
@@ -2,21 +2,24 @@
...
       <p>Total cost: <%= number_to_currency(@order.total_price) %></p>
-      <p>Purchased on: <%= @order.created_at %></p>
   <table class="table table-striped table-bordered text-left">
     <tr>
       <th></th>
       <th>Item</th>
+      <th>Price</th>
       <th>Quantity</th>
-      <th>Cost</th>
+      <th>Subtotal</th>
```
Comment (snayrouz) [inline app/views/orders/show.html.erb:14]: thanks for correcting this

File: app/views/orders/show.html.erb [hunk 4]
```diff
@@ -2,21 +2,24 @@
...
     </tr>
   <% @order_items.each do |order_item| %>
     <tr>
       <td><%= image_tag(order_item.item.image, class: 'thumbs') %></td>
-      <td><%= order_item.item.title %></td>
+      <td><%= link_to order_item.item.title, item_path(order_item.item) %></td>
+      <td><%= number_to_currency(order_item.item.price) %></td>
       <td><%= order_item.quantity %></td>
       <td><%= number_to_currency(order_item.line_item_total) %></td>
```
Comment (snayrouz) [inline app/views/orders/show.html.erb:22]: sweet

**Pull request merged**